src/core/Global.ttslua

```diff
@@ -1014,14 +1014,17 @@ function titleSplash(scenarioName)
       Wait.stop(hideTitleSplashWaitFunctionId)
       hideTitleSplashWaitFunctionId = nil
       UI.setAttribute('title_splash', 'active', false)
+      -- UI.setAttribute('title_gradient', 'active', false)
     end
 
     -- display scenario name and set a 4 seconds (2 seconds animation and 2 seconds on screen)
     -- wait timer to hide the scenario name
-    UI.setValue('title_splash', scenarioName)
+    UI.setValue('title_splash_text', scenarioName)
     UI.show('title_splash')
+    -- UI.show('title_gradient')
     hideTitleSplashWaitFunctionId = Wait.time(function()
       UI.hide('title_splash')
+      -- UI.hide('title_gradient')
```

Comment:
What's the reasoning for the commented out lines with `title_gradient`? Should they be here?